### ISSUE TITLE
[BUG] Fan detail page fails to load

### ISSUE THREAD

**Issue by 0x414c49** [joBr99/nspanel-lovelace-ui#417]
### PROBLEM DESCRIPTION
(esphome-based fan) When I open the details page, the page **freezes**, and I can't do anything, for example, the on/off button stays "off" (the fan is on), and it won't show me the slider.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Provide your apps.yaml:
```
      - type: cardEntities
        title: Air
        entities:
          - entity: fan.open_duco
            name: Ventilator
          - entity: sensor.duco_humidity
            name: Humidity
            icon: thermometer-water
          - entity: sensor.bedroom_air_purifier_air_quality
            name: Air Quality
            icon: molecule-co2


```
- [x] Go to Settings>Add-ons>AppDaemon>Log and then, provide the output of the log during your issue/bug occurs:

In the log below you can see, it loaded the details page I have and when I clicked on the open_duco entity on screen, the log of unable to decode MQTT message appeared. 
```
entityUpd~Air~1|1~fan~fan.open_duco~���~63142~Ventilator~1~text~sensor.duco_humidity~���~17299~Humidity~46.5 %~text~sensor.bedroom_air_purifier_air_quality~���~17299~Air Quality~excellent 
2022-08-27 15:45:15.645970 INFO MQTT: Unable to decode MQTT message

// log from tasmota console

14:45:15.778 MQT: tele/tasmota_7DE400/RESULT = {"CustomRecv":"event,pageOpenDetail,popupFan,fan.open_ducoK#����"}
14:45:25.839 NXP: Received Raw = bytes('55BB23006576656E742C627574746F6E5072657373322C706F7075704C696768...')
14:45:25.857 MQT: tele/tasmota_7DE400/RESULT = {"CustomRecv":"event,buttonPress2,popupLight,bExit"}
14:45:26.009 SRC: MQTT

```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_
EU and latest firmware version.

**Comment by joBr99** [comment]
this is really strange,

can you rename `fan.open_duco` in homeassistant to `fan.open_ducox` and check if this issue persists?

**Comment by 0x414c49** [comment]
"can you rename fan.open_duco in homeassistant to fan.open_ducox and check if this issue persists?"

Before you say something, I renamed it from "open-duco" in HomeAssistant to ventilator the result still is the same.

**Comment by joBr99** [comment]
Flash the lastest dev version and check again

`FlashNextion http://nspanel.pky.eu/lui.tft`

**Comment by 0x414c49** [comment]
> Flash the lastest dev version and check again
> 
> `FlashNextion http://nspanel.pky.eu/lui.tft`

**Solved the issue. Did you change anything in the Nextion?** Although there is one small issue, the fan detail page shows a lamp icon on top instead of a fan.

**Comment by joBr99** [comment]
yeah, I did change something after the commands sending this serial message, Nextion is so buggy ...

I will check the icon later, could be firmware or appdaemon releated

**Comment by 0x414c49** [comment]
Thanks Johannes.  I hate Nextion, and I just don't know how you could make this possible with that!

Off topic:
I'm not sure if you saw NSPanel Pro; that one has a real "LCD" screen and Android inside; I'm not sure if anyone can do a custom firmware release for that.

**Comment by joBr99** [comment]
> Thanks Johannes. I hate Nextion, and I just don't know how you could make this possible with that!

yeah it is definitely not suided for an application like this ... nextion is the maintainability hell


I thought of buying one, but after seeing that it doesn't have a relay I did not. It is a low powered Android 8.1 device, stock software is garbage.

> Just got the sonoff NS pro
> trivial to root 😄
> managed to get the HA apk on it and performance is ... OK
> but eventually out of memory / crashes
> it's another one of those cheap android 8.1 devices with 2 gig ram

Seems to be rootable, but to acually use it I guess a standard linux instaed of android would be better, which might also be possible somewhere in future, but do not buy this device if you are hoping for somebody to do this in future.

**Comment by 0x414c49** [comment]
> I thought of buying one, but after seeing that it doesn't have a relay I did not. It is a low powered Android 8.1 device, stock software is garbage.
I'm not planning to buy one, NSPanel with your Firmware is awesome and at the moment is beyond my needs. The moment I got NSPanel, I was trying to hack around Nexiton but hell no, their editor is just garbage, You are a hero!

Where did you get that message about NS Pro? if there is path to root, I might try to install a lightweight Linux as my daily failure hacks :D

**Comment by joBr99** [comment]
join unoffical nextion discord server

**Comment by joBr99** [comment]
Can you update your app in hacs to main and restart appdaemon?

I don't have that issue with the lightbulb, so it might be already fixed.

https://docs.nspanel.pky.eu/faq/#how-to-upgrade-from-a-release-to-the-current-development-version

**Comment by joBr99** [comment]
lightbulb was fixed in #391
